resources/scss/ceres/common/_common.scss

```diff
@@ -1,22 +1,26 @@
-.center-xy {
-    display: flex;
-    flex-flow: row nowrap;
-    justify-content: center; // center on the x axys
-
-    img,
-    .owl-stage-outer {
-        align-self: center; // center on the y axys
-    }
-}
-
-.not-found-text {
-    font-size: 115px;
-    text-shadow: 4px 3px 0 $gray-100, 9px 8px 0 $gray-300;
-}
-
-// TODO: resolve usages to .container
-.container-max {
-    @extend .container-fluid;
-    z-index: auto;
-    max-width: 1200px;
-}
+.center-xy {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: center; // center on the x axys
+
+    img,
+    .owl-stage-outer {
+        align-self: center; // center on the y axys
+    }
+}
+
+.not-found-text {
+    font-size: 115px;
+    text-shadow: 4px 3px 0 $gray-100, 9px 8px 0 $gray-300;
+}
+
+// TODO: resolve usages to .container
+.container-max {
+    @extend .container-fluid;
+    z-index: auto;
+    max-width: 1200px;
+}
+
+.cursor-pointer {

```

Comment:
brauchen wir die Klasse wirklich? Oft ist es eher ein Indiz für falsch verwendete Elemente, wenn kein Cursor gesetzt wird. In dem PR sehe ich z.B. nur den ToTop-Button mit der Klasse. Wenn wir hier einen echten Button verwenden, sollte der Cursor autom. korrekt sein.

Reply:
done